app/src/ui/lib/dark-theme.ts

```diff
@@ -14,7 +14,5 @@ export function isDarkModeEnabled() {
     return false
   }
 
-  // remote is an IPC call, so if we know there's no point making this call
-  // we should avoid paying the IPC tax
-  return remote.systemPreferences.isDarkMode()
+  return nativeTheme.shouldUseDarkColors
```

Comment:
Reading that documentation it would appear to me as if the `nativeTheme` module is only available in the `main` process. I believe this will fail at runtime